.drone.yml

```diff
@@ -143,9 +144,12 @@ trigger:
   - pull_request
   paths:
     exclude:
-    - '*.md'
     - docs/**
-    - latest.json
+    include:
+    - pkg/**
+    - packaging/*
+    - .drone.yml
+    - conf/*
```

Comment:
Should probably include go.mod/sum changes. Exclude latest.json and all *.md files?